packages/dev/core/src/Inputs/scene.inputManager.ts

```diff
@@ -129,6 +130,11 @@ export class InputManager {
      * @returns Mesh that the pointer is pointer is hovering over
      */
     public get meshUnderPointer(): Nullable<AbstractMesh> {
+        if (this._movePointerInfo) {

```

Comment:
this looks really suspicous, I wonder if the code could be more explicit ?

Reply:
Changed to use more explicit function call